swagger.yaml

```diff
@@ -1682,6 +1828,63 @@ components:
         - START_TO_START
         - FINISH_TO_FINISH
 
+    Notification:
+      type: object
+      description: Notification object that is created upon an event and sent to multiple users
+      required:
+        - message
+        - type
```

Comment:
If notifications are task specific, should taskId be required? Or do you think it is fine as optional for future flexibility of having other notification types other than task related?

Reply:
I am not sure it's always applicable to all the types? If it's notification about task deletion for instance then we will have no valid task id at the point when notification is read